projects/okernel/Dockerfile.okernel

```diff
@@ -1,4 +1,4 @@
-FROM linuxkit/alpine-build-kernel:cfdd576c36a52ed2dd62f237f79eeedc2dd3697b@sha256:3fe08db373a9373ba1616a485858f01ebd2d7a3cb364a099d0ed8b45fa419da2
+FROM linuxkit/kernel-compile:1b396c221af673757703258159ddc8539843b02b@sha256:6b32d205bfc6407568324337b707d195d027328dbfec554428ea93e7b0a8299b AS kernel-build
```

Comment:
`linuxkit/kernel-compile` is no longer a thing. It would be better to replicate what's here https://github.com/linuxkit/linuxkit/blob/master/kernel/Dockerfile#L1

or probably https://github.com/linuxkit/linuxkit/blob/master/kernel/Dockerfile.kconfig#L2 might be sufficient